Add support for featured and high quality apps (#677)
YunoHost/apps: hq_validation_template.md

# Validation template for High Quality tag request

This template is designed to be used as it is by Apps group to validate requests from packagers for the tag High Quality.

Mandatory check boxes:
- [ ] The package is level 7.

> - [ ] The package is level 7 since at least 2 months.

Reviewer: ```suggestion
- [ ] The package has been level 7 for at least 2 months.
```

> - [ ] The package has been level 7 for at least 2 months.

Reviewer 2: Idk ¯\\\_(ツ)\_/¯

```suggestion
- [ ] The package has been level 7 for at least 1 month.
```

Author: So the app should be in the community for at least 2 month, but level 7 for only 1 ?

Reviewer 2: Yup ?

Author: Well... Why not...
Anyway, we'll see how it's going and we could change it later.

> - [ ] The package is in the list since at least 2 months.

Reviewer: ```suggestion
- [ ] The package has been in the list for at least 2 months.
```

- [ ] The package is up to date regarding the packaging recommendations and helpers.
- [ ] The repository has a testing branch.
- [ ] All commits are made in testing branch before being merged into master.

> - [ ] The list point to HEAD, not a specific commit.

Reviewer: ```suggestion
- [ ] The list points to HEAD, not to a specific commit.
```
